File: JAVASCRIPT/day 10/q2.js
console.log("----------Question-2----------")

function outer(){

    let a=8; 
    let b=5

    function inner(){
        let num1=`The number ${a} is =${a**2}`
        let num2=`The number ${b} is =${b**2}`
        return[num1,num2]
    }

    return inner
}

let result=outer();
console.log(result())

File: C LANGUAGE/project 4/q2.c
#include <stdio.h>

int main(){

    int number=11;;

    for (int i=1; i<=4; i++)
    {
        for (int j=1; j<=i; j++)
        {
            printf("%d ", number);
            number++;
        }
        printf("\n");
    }
    
    

    return 0;
}